docs/docs/gatsby-link.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Gatsby Link API
+title: API Gatsby Link
 ---
 
-For internal navigation, Gatsby includes a built-in `<Link>` component as well as a `navigate` function which is used for programmatic navigation.
+Do nawigacji wewnątrz strony, Gatsby dostarcza wbudowany komponent `<Link>` oraz funkcję `navigate` służącą do nawigacji programowej.
 
-Gatsby's `<Link>` component enables linking to internal pages as well as a powerful performance feature called preloading. Preloading is used to prefetch resources so that the resources are fetched by the time the user navigates with this component. We use an `IntersectionObserver` to fetch a low-priority request when the `Link` is in the viewport and then use an `onMouseOver` event to trigger a high-priority request when it is likely that a user will navigate to the requested resource.
+Komponent `<Link>` umożliwia linkowanie do stron wewnętrznych oraz korzystanie z ładowania wstępnego (_preloading_). Ładowanie wstępne wykorzystywane jest do ładowania zasobów strony, zanim użytkownik na nią przejdzie, co zwiększa wydajność i ogólną responsywność strony. W tym celu używany jest `IntersectionObserver`, który wstępnie wykonuje zapytanie z niskim priorytetem gdy `Link` znajdzie się w polu widzenia użytkownika, a następnie wykonuje zapytanie o wyższym priorytecie gdy uruchomiony zostanie event `onMouseOver` elementu `Link` - w tym momencie użytkownik najprawdopodobniej będzie chciał przejść na stronę, na której pobierane zasoby będą się znajdować.
 
-The component is a wrapper around [@reach/router's Link component](https://reach.tech/router/api/Link) that adds useful enhancements specific to Gatsby. All props are passed through to @reach/router's `Link` component.
+Komponent `Link` zbudowany jest na bazie [komponentu link z @reach/router](https://reach.tech/router/api/Link), dodając do niego użyteczne usprawnienia specyficzne dla Gatsby. Wszystkie właściwości przekazywane są do komponentu `Link` z @reach/router.
```

Comment:
Dla całego PR: nie tłumaczyłbym props na właściwości. Lepiej zostawić to w oryginalnej nazwie, ewentualnie odmieniac.
```suggestion
Komponent `Link` zbudowany jest na bazie [komponentu link z @reach/router](https://reach.tech/router/api/Link), dodając do niego użyteczne usprawnienia specyficzne dla Gatsby. Wszystkie propsy przekazywane są do komponentu `Link` z @reach/router.
```

Reply:
Warto by to dodać do "słowniczka" w style-guide.md, bo np. dokumentacja Reacta używa słowa "właściwości" :)